src/targets/mas_steekproef_zavelberg/output/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!.gitignore
+!*.*
```

Comment:
Can you simply remove this `.gitignore` file to get the same effect?

Reply:
added `fs::dir_create("output")`